style-sheets/primevue-components-overrides.scss

```diff
@@ -391,7 +392,57 @@ label {
 .#{variables.$primevue-prefix}-card .#{variables.$primevue-prefix}-card-footer {
     padding: 0;
 }
+
 /* ------------------------------------ primevue table --------------------------------- */
+
+.custom-access-table {
```

Comment:
Hi Jota, we don't put any custom class override in this file. These style change actually apply to the primevue data table component, and the class name for primevue data table component is ".p-data-table". 
<img width="1407" alt="Screen Shot 2023-08-24 at 11 39 54 AM" src="https://github.com/bcgov/nr-fsa-theme/assets/23131735/71505d34-0113-4221-a5f2-62e609615f59">